### PR TITLE
Buff wander rate

### PR DESCRIPTION
**Pull request by HLXII**
By my calculations Starf with maxed out farm should have a 10% chance every 24 hours.

### CODE REVIEW

File: src/modules/GameConstants.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const AMAZE_MULCH_GROWTH_MULTIPLIER = 1.25;
 export const AMAZE_MULCH_PRODUCE_MULTIPLIER = 1.5;
 export const AMAZE_MULCH_MUTATE_MULTIPLIER = 1.25;
 
-export const WANDER_RATE = 0.00001;
+export const WANDER_RATE = 0.0005;
```
Comment (RedSparr0w) [inline src/modules/GameConstants.ts:71]: This seems like quite a large increase, 50x larger than before?

Comment (HLXII) [reply src/modules/GameConstants.ts:71]: I balanced it with Starf in mind.

Originally with .00001 chance, there was a .2% chance of getting Starf every 24 hours. Buffed it up to 10%.

Comment (RedSparr0w) [reply src/modules/GameConstants.ts:71]: Could we not just boost the Starf mutation rate instead of the wondering Pokémon chance?

Comment (HLXII) [reply src/modules/GameConstants.ts:71]: The starf mutation always occurs on wandering shiny pokemon. There isn't a mutation rate for that. To increase the chances of obtaining a Starf plant, we can either buff the wander rate, or buff the shiny chances on the farm.

**Review by RedSparr0w — APPROVED** [review]

**Pull request merged**